src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,14 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+	
+	public default String getPrintableString(Printable... printables){
```

Comment:
When a subclass calls `getPrintableString()`, it seems that it won't print anything out.
I think you still need to implement the interface for each subclass.